java-runtime/build.gradle.kts

```diff
@@ -42,6 +42,10 @@ plugins {
     module
 }
 
+apply {
+    plugin("io.spine.mc-java")
```

Comment:
This plugin complicates matters in terms of the required dependencies for the code it is going to generate. Let's avoid.